templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -94,12 +94,33 @@ contents:
         nmcli c add type ovs-port conn.interface br-ex master br-ex con-name ovs-port-br-ex
       fi
 
+      extra_phys_args=""
+      # check if this interface is a vlan, bond, or ethernet type
+      if nmcli --fields connection.type conn show ${old_conn} | grep -q vlan; then
```

Comment:
if [[ $(nmcli --get-values connection.type conn show ${old_conn}) == "vlan" ]]; then

? Especially, then it won't be fooled by `connection.type: macvlan`

(Likewise with `--get-values` below)

Reply:
ah didn't know about --get-values, will change to use that and direct compare. Thanks.